pkgs/development/tools/misc/dura/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgs }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dura";
+  version = "unstable-2022-01-07";
+
+  src = fetchFromGitHub {
+    owner = "tkellogg";
+    repo = "dura";
+    rev = "a41863879772650f2538a978ca926357bae674db";
+    sha256 = "sha256-eBgD1rBSm314I9/f1q/tOzJHjMrftbzow2HTz37njak=";
+  };
+
+  cargoSha256 = "sha256-pBi+6QfFxGjNJ+13YnMUDpDIGTtRoJhEGazrpKb04Cg=";
+
+  buildInputs = with pkgs; [
+    openssl
+  ];
+
+  nativeBuildInputs = with pkgs; [
+    pkg-config
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "You shouldn't ever lose your work if you're using Git.";
```

Comment:
1. Short `description`s should not end with period.
2. A better desc would be `A background process that saves uncommited changes on git repos`.

Reply:
I fixed this on @AndersonTorres suggestion.

@tkellogg being the creator of Dura, I'm going to ask him some feedback too.

@tkellogg is it ok for you ?

Reply:
yeah that's fine